src/WebUI/Workloads/WorkloadsStore.ts

```diff
@@ -110,10 +130,11 @@ export class WorkloadsStore extends StoreBase {
                 orphanPods.push(pod);
             }
         });
-
-        this._state.orphanPodsList = orphanPods;
-        this.emit(WorkloadsEvents.WorkloadPodsFetchedEvent, this);
-
+        const isEquals = Utils.isDeepEquals(this._state.orphanPodsList || [], orphanPods);
+        if (!isEquals || !this._state.orphanPodsList) {
+            this._state.orphanPodsList = orphanPods;
+            this.emit(WorkloadsEvents.WorkloadPodsFetchedEvent, this);
+        }
         if (this._state.orphanPodsList && this._state.orphanPodsList.length > 0) {
```

Comment:
should not we keep this inside the if condition of "if (!isEquals || !this._state.orphanPodsList) {"?